src/Avalonia.Controls/Button.cs

```diff
@@ -373,7 +419,16 @@ private void CanExecuteChanged(object sender, EventArgs e)
         /// <param name="root">The input root.</param>
         private void ListenForDefault(IInputElement root)
         {
-            root.AddHandler(KeyDownEvent, RootKeyDown);
+            root.AddHandler(KeyDownEvent, RootDefaultKeyDown);
+        }
+
+        /// <summary>
+        /// Starts listening for the Enter key when the button <see cref="IsCancel"/>.
```

Comment:
A little copy-paste problem: Enter -> Escape

Reply:
thanks! I have fixed the typo.

Reply:
No problem. Thanks you! IsCancel - very useful feature :)